custom-config.fcc

```diff
@@ -107,10 +107,12 @@ systemd:
         After=create-datastore.service
         Before=coreos-installer.target
         ConditionKernelCommandLine=custom-config
+        RequiresMountsFor=/boot
 
         [Service]
         Type=oneshot
         ExecStart=/usr/local/bin/capture-macs
+        MountFlags=slave
```

Comment:
@RamLavi can you please explain what this flag doing?

Reply:
[MountFlags](https://www.freedesktop.org/software/systemd/man/systemd.exec.html)=
Takes a mount propagation setting: shared, slave or private, which controls whether file system mount points in the file system namespaces set up for this unit's processes will receive or propagate mounts and unmounts from other file system namespaces. See mount(2) for details on mount propagation, and the three propagation flags in particular.

This setting only controls the final propagation setting in effect on all mount points of the file system namespace created for each process of this unit. Other file system namespacing unit settings (see the discussion in PrivateMounts= above) will implicitly disable mount and unmount propagation from the unit's processes towards the host by changing the propagation setting of all mount points in the unit's file system namespace to slave first. Setting this option to shared does not reestablish propagation in that case.

If not set – but file system namespaces are enabled through another file system namespace unit setting – shared mount propagation is used, but — as mentioned — as slave is applied first, propagation from the unit's processes to the host is still turned off.

It is not recommended to use private mount propagation for units, as this means temporary mounts (such as removable media) of the host will stay mounted and thus indefinitely busy in forked off processes, as unmount propagation events won't be received by the file system namespace of the unit.

Usually, it is best to leave this setting unmodified, and use higher level file system namespacing options instead, in particular PrivateMounts=, see above.

This option is only available for system services and is not supported for services running in per-user instances of the service manager.

Reply:
But looks like this is the default so way do we have to set it? 

Sorry for the questions I just want to make sure that we are not adding things that we will need to keep track on in the future or we will need to change because the _slave_  option doesn't exists anymore and we need to find something else

Reply:
let me check this..

Reply:
You are right, we can remove it, but imho we better be explicit.

Reply:
The default propagation is `shared`, not `slave`. Without `slave`, the `/boot` read-write remount will be visible to all processes, essentially reverting the default read-only stance we ship with.

I wouldn't worry about the option going away.

Reply:
@kobi86 can you check with this option on?